System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"
+
+SET filePath="System Information\System_Information.txt"
+
+:: Run SYSTEMINFO
+echo Running command: systeminfo
+echo ------- Output of command: systeminfo --------- > %filePath%
+systeminfo >> %filePath%
+
+:: Run WMIC
+echo Running command: WMIC
+echo ------- Output of command: WMIC --------- >> %filePath%
+wmic /APPEND:%filePath% product get name,vendor,version >> nul
+
+::Run Dxdiag and append data to System_Information.txt
+echo Running command: dxdiag
+echo ------- Output of command: dxdiag --------- >> %filePath%
+dxdiag /T dxdiag.txt
+type dxdiag.txt >> %filePath%
+del dxdiag.txt
+
+::Run PsInfo and append data to System_Information.txt

```

Comment:
Yes, I know what you mean! The trouble is, Microsoft doesn't allow for PsInfo to be redistributed.

I think the following command should work?

`powershell "Get-ChildItem HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall | ForEach-Object { Get-ItemProperty $_.PsPath } | Format-Table  -AutoSize DisplayName,DisplayVersion,Publisher"`

(all one line)